lib/chef/provisioning/aws_driver/aws_resource.rb

```diff
@@ -5,199 +5,208 @@
 require 'chef/provisioning/aws_driver/aws_taggable'
 
 # Common AWS resource - contains metadata that all AWS resources will need
-module Chef::Provisioning::AWSDriver
-class AWSResource < Chef::Provisioning::AWSDriver::SuperLWRP
-  actions :create, :destroy, :purge, :nothing
-  default_action :create
-
-  def initialize(name, run_context=nil)
-    name = name.public_send(self.class.aws_sdk_class_id) if name.is_a?(self.class.aws_sdk_class)
-    super
-    if run_context
-      driver run_context.chef_provisioning.current_driver
-      chef_server run_context.cheffish.current_chef_server
-    end
-  end
+class Chef
```

Comment:
Github didn't do anything intelligent with this file's diff: is it just indenting?